tools/nxfuse/README_SMARTFS.md

```diff
@@ -1,20 +1,86 @@
-nxfuse
-======
+# How to make smartfs user image for TizenRT
+
+## Contents
+> [How to Enable SmartFS user image](#how-to-enable-smartfs-user-image)  
+> [About NxFuse tool](#about-nxfuse-tool)  
+> [How to use nxfuse?](#how-to-use-nxfuse-?)  
+> [Formatting / Reformatting](#formatting-reformatting)  
+
+## How to Enable SmartFS user image
+SMARTFS file system(CONFIG_FS_SMARTFS) needs to be enabled in order to use SMARTFS image.
+
+Enable FS_SMARTFS config through menuconfig at *os* folder.
+
+```bash
+cd $TIZENRT_BASEDIR/os
+make menuconfig
+```
+
+### Configurations
+Enable below configurations as followings
+
+```
+->BuildSetup
+	->Binary Output Formats
+		-> Enable "Make smartfs binary image" (=y)
+```
+
+```
+->Board Selection
+	-> Adjust smartfs partition size
+```
+
+```
+->File Systems
+	-> Memory Technology Device (MTD) Support
+		-> SMART Device options
+			->  set "SMART Device sector size" = (512 bytes)
+```
+
+```
+->File Systems
+	-> Memory Technology Device (MTD) Support
+		-> SMART Device options
+			->  Enable "Enable Sector CRC error detection" (=y)
+```
+
+### SmartFS Partition size
+Adjust blkcount in $TIZENRT_BASEDIR/tools/nxfuse/mksmartfsimg.sh. blkcount multiplied by blksize should be equal to SMARTFS partition size.
+
+### Content
+Place the user files to be loaded, in path "$TIZENRT_BASEDIR/tools/fs/contents-nxfuse/<CONFIG_ARCH_BOARD>/base-files/" (CONFIG_ARCH_BOARD--> refer os/.config)
+
+For Ex: If "CONFIG_ARCH_BOARD=imxrt1020-evk", then the base-files path would be "$TIZENRT_BASEDIR/tools/fs/contents-nxfuse/imxrt1020-evk/base-files/"
```

Comment:
You set the path of content in script as shown below.
```
CONTENTSDIR=$BASE_DIR/tools/fs/contents-smartfs
```
But here you wrote `contents-nxfuse`.